core/services/relay/evm/mercury/config_poller.go

```diff
@@ -90,33 +90,40 @@ type ConfigPoller struct {
 	destChainLogPoller logpoller.LogPoller
 	addr               common.Address
 	feedId             common.Hash
+	notifyCh           chan struct{}
 }
 
 func FilterName(addr common.Address) string {
 	return logpoller.FilterName("OCR2 Mercury ConfigPoller", addr.String())
 }
 
 // NewConfigPoller creates a new Mercury ConfigPoller
-func NewConfigPoller(lggr logger.Logger, destChainPoller logpoller.LogPoller, addr common.Address, feedId common.Hash) (*ConfigPoller, error) {
+func NewConfigPoller(lggr logger.Logger, destChainPoller logpoller.LogPoller, addr common.Address, feedId common.Hash, eventBroadcaster pg.EventBroadcaster) (*ConfigPoller, error) {
```

Comment:
potential follow up - is there a way to combine mecury/config_poller and evm/config_poller? Look's fairly copy pasta